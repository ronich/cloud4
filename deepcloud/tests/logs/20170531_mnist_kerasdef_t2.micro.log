Downloading data from https://s3.amazonaws.com/img-datasets/mnist.npz
    8192/11490434 [..............................] - ETA: 0s 1531904/11490434 [==>...........................] - ETA: 0s 3465216/11490434 [========>.....................] - ETA: 0s 5521408/11490434 [=============>................] - ETA: 0s 8044544/11490434 [====================>.........] - ETA: 0s10010624/11490434 [=========================>....] - ETA: 0sTrain on 60000 samples, validate on 10000 samples
Epoch 1/12
360s - loss: 10.3837 - acc: 0.3293 - val_loss: 7.3190 - val_acc: 0.5447
Epoch 2/12
346s - loss: 8.8693 - acc: 0.4484 - val_loss: 6.5175 - val_acc: 0.5943
Epoch 3/12
348s - loss: 7.7507 - acc: 0.5179 - val_loss: 6.0471 - val_acc: 0.6242
Epoch 4/12
363s - loss: 7.0763 - acc: 0.5597 - val_loss: 5.4139 - val_acc: 0.6632
Epoch 5/12
376s - loss: 6.4494 - acc: 0.5986 - val_loss: 5.4263 - val_acc: 0.6626
Epoch 6/12
1159s - loss: 6.2218 - acc: 0.6129 - val_loss: 5.0996 - val_acc: 0.6831
Epoch 7/12
3637s - loss: 6.2129 - acc: 0.6135 - val_loss: 4.4115 - val_acc: 0.7253
Epoch 8/12
3596s - loss: 6.0081 - acc: 0.6263 - val_loss: 4.5750 - val_acc: 0.7154
Epoch 9/12
3651s - loss: 5.7200 - acc: 0.6440 - val_loss: 4.6064 - val_acc: 0.7136
Epoch 10/12
3588s - loss: 5.5985 - acc: 0.6517 - val_loss: 3.8099 - val_acc: 0.7622
Epoch 11/12
3654s - loss: 5.3540 - acc: 0.6668 - val_loss: 3.3648 - val_acc: 0.7905
Epoch 12/12
3603s - loss: 5.3372 - acc: 0.6677 - val_loss: 3.7558 - val_acc: 0.7658
