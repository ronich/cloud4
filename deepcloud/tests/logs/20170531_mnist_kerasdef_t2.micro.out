epoch,acc,loss,val_acc,val_loss
0,0.329283333333,10.3837282399,0.5447,7.31904538116
1,0.448383333333,8.86932880376,0.5943,6.51746033554
2,0.517933333365,7.75073766251,0.6242,6.04713542175
3,0.559683333302,7.07631110229,0.6632,5.41386850891
4,0.598566666698,6.44944685364,0.6626,5.42628553085
5,0.612900000032,6.22176688766,0.6831,5.09959273987
6,0.613466666698,6.21285912094,0.7253,4.41153490906
7,0.626333333365,6.00812844671,0.7154,4.5749658905
8,0.643966666667,5.71997265422,0.7136,4.606371698
9,0.651666666635,5.59849172007,0.7622,3.80993780518
10,0.666800000032,5.35404174474,0.7905,3.36484163666
11,0.667733333302,5.33724323196,0.7658,3.755810849
